src/editors/containers/VideoEditor/components/VideoEditorModal.jsx

```diff
@@ -14,9 +13,7 @@ export const {
 export const hooks = {
   initialize: (dispatch, selectedVideoId, selectedVideoUrl) => {
     // eslint-disable-next-line react-hooks/rules-of-hooks
-    React.useEffect(() => {
-      dispatch(thunkActions.video.loadVideoData(selectedVideoId, selectedVideoUrl));
-    }, []);
+    dispatch(thunkActions.video.loadVideoData(selectedVideoId, selectedVideoUrl));
```

Comment:
Nit:
The `rules-of-hooks` disable in L15 can be removed.
If returnToGallery is renamed to useReturnToGallery, the other `rules-of-hooks` disables can be removed as well.